sdv/demo.py

```diff
@@ -322,17 +322,14 @@ def _load_tabular_dummy():
     """Load a dummy tabular demo dataframe."""
     age = np.random.randint(30, 50, 12)
     age_when_joined = age - np.random.randint(0, 10, 12)
-    faker = Faker()
-    names = [faker.name() for _ in range(12)]
-    adresses = [faker.address() for _ in range(12)]
+    employee_ids = np.random.randint(100000, 999999, 12)
```

Comment:
I was going for a 6 digit id but this range is so large I don't think the constraint really will have to do much